Added AU CFP, grants and tickets reminder (#1794)
writethedocs/www: docs/conf/australia/2022/news/tickets-cfp-reminder.rst

:template: {{year}}/generic.html

.. post:: August 29, 2022
   :tags: {{shortcode}}-{{year}}, website, cfp, tickets

Ticket Sales, Grant Program, and Call for Proposals Reminder
=============================================================

Greetings, documentarians! Just a quick reminder that tickets are `on sale now <https://www.writethedocs.org/conf/australia/{{year}}/tickets/>`_, and to remind you that our `Call for Proposals <https://www.writethedocs.org/conf/australia/{{year}}/cfp/>`_ is still open.

Following the success of our previous virtual conferences, we look forward to more opportunities to bring the community together virtually. We excited to share the wonderful energy of our community wherever you are in the world.

If there is something you’d really like to see a talk about this year, submit a proposal for it or refer someone else who would be good!

Get your ticket for the conference
-----------------------------------

**The Australia conference will be held online on {{ date.short }}.** The `full event website <https://www.writethedocs.org/conf/australia/{{year}}/>`_ is live, with some preliminary details, and we'll be filling in more as it gets closer.

> Tickets are on sale now, so go ahead and `register for your ticket now <https://www.writethedocs.org/conf/australia/{{year}}/tickets/>`_! Even though our capacity is more flexible due to the virtual event platform, we might still sell out to make sure we're providing a manageable event size for our attendees, as always.

Reviewer: ```suggestion
Tickets are already on sale, so go ahead and `register for your ticket now <https://www.writethedocs.org/conf/australia/{{year}}/tickets/>`_!
```

> Similar to our previous years' virtual conference, we're going to keep as much as possible of the spirit of the event alive:

Reviewer: ```suggestion
We bring as much of the spirit of our in person events as we can to  our virtual conference, including but not limited to:
```
